scripts/update_teams.py

```diff
@@ -107,9 +83,7 @@ def create_team(org, name, description, repos):
     env = jinja2.Environment(undefined=NullUndefined)
 
     with open(recipe) as fh:
-        environ = SilentDict()
-        environ.undefined = ""
-        contents = env.from_string(''.join(fh)).render(environ=environ)
+        contents = env.from_string(''.join(fh)).render()
```

Comment:
@jakirkham - we *really* need to stop using ``environ``. It is the wrong solution to the problem.
The whole VC thing is a mess, and it is driving us down a path we shouldn't be taking. Looking forwards to finding a more long-term solution this week 😄